docs/advanced-topics/legacy/wf-legacy-registration.rst

```diff
@@ -0,0 +1,88 @@
+Legacy Workflow Registration Methods
+====================================
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. include:: /getting-started/github-apps/note--legacy-dont-sync.rst
+
+If you would prefer not to use GitHub App registration for your workflows, you can instead use one of these methods.
+
+Traditional Registration
```

Comment:
Should we still say Traditional or Legacy here? Should this also be listed as Option A, B, C?